wolfcrypt/src/aes.c

```diff
@@ -10382,11 +10382,11 @@ int wc_AesOfbDecrypt(Aes* aes, byte* out, const byte* in, word32 sz)
 /* Initialize key wrap counter with value */
 static WC_INLINE void InitKeyWrapCounter(byte* inOutCtr, word32 value)
 {
-    int i;
+    word32 i;
     word32 bytes;
 
     bytes = sizeof(word32);
-    for (i = 0; i < (int)sizeof(word32); i++) {
+    for (i = 0; i < sizeof(word32); i++) {
```

Comment:
I made `i` a `word32` because everything else in the function was already unsigned, so it just seemed weird for `i` to be an outlier there throwing several `-Wconversion`s.